backend/packages/Upgrade/.env.example

```diff
@@ -63,7 +63,7 @@ MONITOR_PASSWORD=1234
 #
 # Google Client Id
 #
-GOOGLE_CLIENT_ID=google_project_id
+GOOGLE_CLIENT_ID=135765367152-pq4jhd3gra10jda9l6bpnmu9gqt48tup.apps.googleusercontent.com
```

Comment:
.env.example shouldn't need to be changed, it's just a placeholder

Reply:
I think we can close this PR as we have diff client ids anyway and just add this step in our setup doc.